corporate/tests/test_support_views.py

```diff
@@ -757,7 +757,7 @@ def create_customer_and_plan(self, realm: Realm, monthly: bool = False) -> Custo
     def test_realm_support_view_queries(self) -> None:
         iago = self.example_user("iago")
         self.login_user(iago)
-        with self.assert_database_query_count(18):
+        with self.assert_database_query_count(57):
```

Comment:
Looks like this adds a lot of queries, is it fine for this page or do we need it to be more efficient?

Reply:
This needs to be more efficient. These pages are already very slow enough to often time out; tripling the query count is going to hurt.

Reply:
Okay, found out that we can use `get_recursive_group_members` here, which adds 2 queries.